### DISCUSSION TITLE
Make "Music recording equipment" either obsolete or worth it.

### DISCUSSION THREAD

**Discussion by Lamandus** [cataclysmbn/Cataclysm-BN#3242] [General]
### Is your feature request related to a problem? Please describe.

We have some rooms filled with music equipment. Be it some special xlr cables or microphones we can't use in any shape or form.

They are just clutter and serve absolutly no purpose. Another remain of the DDA times were adding stuff just for clutter is okey.

### Describe the solution you'd like

We have 3 options here.

1. Either we make the cable worth something (uncrafting)
2. Replace those special cables with regular ones (for example copper wire) with migration.
3. Remove those rooms/cables/microphones completly.

### Additional context

![grafik](https://user-images.githubusercontent.com/33199510/216292338-86b7cb58-adfe-49d1-a12b-283ec6896d6b.png)

**Comment by chaosvolt** [comment]
They can't already be uncrafted? Figures they'd overlook something like that, that'd probably be a good start to make them useful for supplies. Making them involved in the recipe for the stereo system would be another minor improvement.

Isn't there also a loudspeaker monster that spawns in certain map extras as a distraction, or is that currently only in DDA? Grabbing it and making a deployable version the player can craft could be an additional option.

**Comment by MochaModifier** [comment]
I don't think these have any real use in DDA either, including other equipment like the mixers.
Could be able to break them down into copper wire and scrap, maybe?

**Comment by chaosvolt** [comment]
> I don't think these have any real use in DDA either, including other equipment like the mixers.

Pretty much a given that they'd be flavor items, yeah. Should be simple enough to add uncrafts, might tinker with it myself when I get back home later today if no one else has, will have to see.
